src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/FipsAwareEncryptedXml.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Xml;
+using System.Xml;
+using System.Linq;
+
+namespace System.Configuration
+{
+    //
+    // Extends EncryptedXml to use FIPS-certified symmetric algorithm
+    //
+    internal sealed class FipsAwareEncryptedXml : EncryptedXml
+    {
+        public FipsAwareEncryptedXml(XmlDocument doc)
+            : base(doc)
+        {
+        }
+
+        public override SymmetricAlgorithm GetDecryptionKey(EncryptedData encryptedData, string symmetricAlgorithmUri)
+        {
+            if (IsAES256(encryptedData, symmetricAlgorithmUri))
+            {
+                // Obtain the EncryptedKey
+                EncryptedKey ek = encryptedData.KeyInfo.OfType<KeyInfoEncryptedKey>().FirstOrDefault()?.EncryptedKey;
+
+                // Got an EncryptedKey, decrypt it to get the AES key
+                if (ek != null)
+                {
+                    byte[] key = DecryptEncryptedKey(ek);
+
+                    // Construct FIPS-certified AES provider
+                    if (key != null)
+                    {
+                        AesCryptoServiceProvider aes = new AesCryptoServiceProvider();
```

Comment:
Always use Aes.Create() on .NET (nee Core).  It's always as FIPSy as can be.

Reply:
What is the proper spilling for it?
I tried that variant, yet it was not being built. 
`Aes.Create(typeof(AesManaged).FullName), typeof(AesManaged)`

Reply:
```suggestion
                        Aes aes = Aes.Create();
```